bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/team/BedWarsTeam.java

```diff
@@ -362,6 +362,7 @@ public void respawnMember(@NotNull Player p) {
         }, 8L);
 
         nms.sendTitle(p, getMsg(p, Messages.PLAYER_DIE_RESPAWNED_TITLE), "", 0, 20, 10);
+        p.sendMessage(getMsg(p, Messages.PLAYER_DIE_RESPAWNED_TEXT));
```

Comment:
I saw you added PLAYER_DIE_RESPAWN_CHAT to all language files but its already added.

Reply:
I see what happend btw. You put TEXT here but in the lang files you put CHAT